src/get_maintainer_wrapper.sh

```diff
@@ -47,6 +47,31 @@ function print_files_authors()
     done
 }
 
+function is_linux_tree_root()
```

Comment:
how about put this function on "commons.sh"? This can be useful for a new version of the function handler for checkpatch (see #40)

Reply:
Seems good! But don't you think it would be nicer to leave the function here, for now, and move it in a PR for #40 ? (Just thinking about that idea you told us of "Do what is sufficient for now, not for the future", you know?)